app/models/procedure.rb

```diff
@@ -515,6 +516,7 @@ def clone(admin, from_library)
     procedure.closing_notification_en_cours = false
     procedure.template = false
     procedure.monavis_embed = nil
+    procedure.procedure_labels = procedure_labels
```

Comment:
```suggestion
    procedure.procedure_labels = procedure_labels&.dup
```
sinon ça va garder les mêmes références